shared/common-adapters/tab-bar.native.js

```diff
@@ -85,10 +78,10 @@ class TabBar extends Component {
 
   _labels (): Array<React$Element> {
     // TODO: Not sure why I have to wrap the child in a box, but otherwise touches won't work
-    return (this.props.children || []).map((item, i) => (
-      <TouchableWithoutFeedback key={item.props.label || i} onPress={item.props.onClick || (() => {})}>
-        <Box style={item.props.containerStyle}>
-          {item.props.tabBarButton || <SimpleTabBarButton tabWidth={this.props.tabWidth} label={item.props.label} selected={item.props.selected} underlined={this.props.underlined} />}
+    return (this.props.children || []).map((item: {props: ItemProps}, i) => (
+      <TouchableWithoutFeedback key={`${i}-${item.props.label}`} onPress={item.props.onClick || (() => {})}>
```

Comment:
My initial thoughts were to allow for repeated labels (don't know what the current value in those would be). That doesn't seem like a necessary use case so I'll remove them.

An issue with using labels as the key is related that we may have tabs with updating labels such as "Followers (8)" becoming "Followers (9)" or something along those lines. Maybe a key should be explicitly provided to the TabBarItems.